packages/ubuntu_desktop_installer/lib/pages/installation_complete/version_extractor.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+class VersionExtractor {
+  String _dummy() {
+    return 'Ubuntu 20.04';
```

Comment:
Instead of a hardcoded string, you should test for the existence of `/etc/os-release` and use the value of the `PRETTY_NAME` key, if it's present. As a last resort, I would just hardcode "Ubuntu", without a version number.